### ISSUE TITLE
Conduct load times out after successfully loading a bundle to the remote server.

### ISSUE THREAD

**Issue by longmuir** [typesafehub/conductr-cli#214]
CLI version 0.53 on an Ubuntu16 laptop.
```
rjkj@rjkj-dev ~/code/soco-akka/file_processor/target/bundle $ conduct load --host tdclddlnva017.ebiz.verizon.com soco-file_processor-v0-aaf237628f870254e90a936b31d813a8c9285b55b7b8ff302ae78af76608e523.zip kafka_config.sh-e9ee0cde04b501c784f0a892857f914a9159b7cb7dfddac3c28c147070707a35.zip
Retrieving bundle...
Retrieving file:///home/rjkj/code/soco-akka/file_processor/target/bundle/soco-file_processor-v0-aaf237628f870254e90a936b31d813a8c9285b55b7b8ff302ae78af76608e523.zip
Retrieving configuration...
Retrieving file:///home/rjkj/code/soco-akka/file_processor/target/bundle/kafka_config.sh-e9ee0cde04b501c784f0a892857f914a9159b7cb7dfddac3c28c147070707a35.zip
Loading bundle to ConductR...
[#################################################] 100%
Bundle aaf237628f870254e90a936b31d813a8-e9ee0cde04b501c784f0a892857f914a waiting to be installed
Error: Timed out: Bundle aaf237628f870254e90a936b31d813a8-e9ee0cde04b501c784f0a892857f914a waiting to be installed
rjkj@rjkj-dev ~/code/soco-akka/file_processor/target/bundle $

```

We were able to start the bundle successfully after this.

**Comment by huntc** [comment]
As an alternate strategy we should consider:
1. Connect to the SSE endpoint but don't start reading ie just open it. 
2. Get the current state via /bundles
3. Once we have the reply for 2, and if then reply does not indicate a load as having completed, start reading the events from the channel opened by 1.

The difference with this strategy is that events will buffer in the background while we perform 2. Buffering happens at the TCP level.

**Comment by huntc** [comment]
@longmuir Can you please confirm you're using the correct version:

```
conduct version
```

The output above doesn't correlate with what we'd expect in the latest release.

**Comment by longmuir** [comment]
The above was captured on the client's screen right after he upgraded the CLI. That said, I've had issues updating the CLI myself with pip3 (like just now :() - where it doesn't get upgrade successfully by pip, so I wouldn't guarantee that it's correct.

**Comment by huntc** [comment]
@longmuir can you please confirm if this issue remains

**Comment by huntc** [comment]
This issue is resolved in ConductR 2.0.
